test/api/v3/integration/tasks/PUT-tasks_id.test.js

```diff
@@ -176,6 +177,42 @@ describe('PUT /tasks/:id', () => {
       expect(savedDaily.reminders[0].id).to.equal(id1);
       expect(savedDaily.reminders[1].id).to.equal(id2);
     });
+
+    it('can set a shortName if the shortName does not already exist', async () => {
+      let savedDaily = await user.put(`/tasks/${daily._id}`, {
+        shortName: 'short-name',
+      });
+
+      expect(savedDaily.shortName).to.eql('short-name');
+    });
+
+    it('ignores shortName if it is already set', async () => {
```

Comment:
But changing the short name would be _completely_ in your control. It's not like you would have to keep an eye on it to make sure it _didn't_ change. You would have to complete the action to make it change.

Here's the use case I'm thinking of.

Let's say I use [GitHabit](http://habitica.wikia.com/wiki/GitHabit_-_GitHub_Webhooks) to score a habit I've given the shortName `github` every time I push commits to Github. This is great. But let's say that at some point in the future, I want to add my Gitlab and Bitbucket accounts to do the same thing on the same habit. I go to try and change the shortName to `git` or `git-score` or something, but I cannot. I now have 2 choices:

1) Delete `github` entirely and create a new habit, thus losing my history
2) Continue using `github` even though it doesn't accurately reflect what it encompasses 

All this because my needs changed slightly.

I don't think it hurts anything to allow the user to change it, especially if we warn them of the consequences before doing so.